spec/features/hq/user_is_creating_multiple_sponsors_spec.rb

```diff
@@ -16,18 +16,11 @@
   end
 end
 
+
 def an_agent_exists
   FactoryGirl.create :agent
 end
 
-def i_sign_in_as_admin
```

Comment:
just moved to `spec/features/hq/admin_user_authentication_spec.rb` together with all auth methods